core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJobTracker.java

```diff
@@ -63,14 +70,18 @@ public void run() {
 							.from(RemoteJobEntry.class)
 							.where(
 								Expressions.builder()
-									.filter(RemoteJobEntry.Expressions.deleted(true))
-									.filter(RemoteJobEntry.Expressions.done())
+									.must(RemoteJobEntry.Expressions.done())
+									.must(Expressions.builder()
```

Comment:
`filter` is more efficient than `must`; the semantics are the same, but it will not score documents based on the sub-expressions, and we don't need to keep score here.

Reply:
@nagyo pointed out that `must` clauses turn into `filter`s within `EsQueryBuilder` if there are no text matching queries or script scoring (in which case the `needsScoring` variable in line 280 in the snippet remains `false`): https://github.com/b2ihealthcare/snow-owl/blob/311c96ec555a64e7762c8dd34c76ce0b35e66ee3/commons/com.b2international.index/src/com/b2international/index/es/query/EsQueryBuilder.java#L232-L283

Reply:
https://github.com/b2ihealthcare/snow-owl/blob/246460bb004abb6caaa6f3d474fbf46c659166ca/commons/com.b2international.index/src/com/b2international/index/es/query/EsQueryBuilder.java#L180-L185

This piece does the magic of converting musts to filters, in case no score is needed, so the end result will be a filter 🎉